rclcpp/src/rclcpp/node.cpp

```diff
@@ -331,7 +331,7 @@ Node::remove_on_set_parameters_callback(const OnSetParametersCallbackHandle * co
 rclcpp::Node::OnParametersSetCallbackType
 Node::set_on_parameters_set_callback(rclcpp::Node::OnParametersSetCallbackType callback)
 {
-  return node_parameters_->set_on_parameters_set_callback(callback);
+  node_parameters_->set_on_parameters_set_callback(callback);
```

Comment:
This change should be reverted. I guess it was accidental.

Reply:
Also, even though the return could probably be removed from L328, it would be better to make the change separately to keep this PR focused on `rclcpp_lifecycle`.